src/test/kotlin/no/nav/familie/ef/mottak/service/SøknadTreeWalkerTest.kt

```diff
@@ -17,7 +17,6 @@ class SøknadTreeWalkerTest {
         assertThat(list).contains("giftIUtlandetDokumentasjon",
```

Comment:
Trenger vi denne testen?

Reply:
Jeg stemmer for å slette den!

Reply:
Var vel kanskje viktigst i forbindelse med utvikling.